index.bs

```diff
@@ -886,6 +886,12 @@ To send a presentation message, the controller or receiver may send a
 : message
 :: The presentation message data.
 
+NOTE: An OSP agent should minimize the latency between applications sending or
+receiving messages and the underlying QUIC connection, as messages may be used
```

Comment:
Is this specifically referring to having performant message serialization and deserialization? It's unclear to me what exactly the steps are between sending a message and it reaching the QUIC API for message sending.

Reply:
Serialization is one necessary step; but basically it's telling the implementation that messages need to be as "real-time" as possible, so don't add unnecessary message buffering in the OSP implementation on top of network buffering in QUIC.

I will add some more detail here along these lines.